heka/files/toml/output/http.toml

```diff
@@ -7,14 +7,16 @@ address = "{{ output.address }}"
 username = "{{ output.username }}"
 password = "{{ output.password }}"
 {%- endif %}
-http_timeout = {{ output.timeout }}
-method = "POST"
+http_timeout = {{ output.http_timeout|default(2000) }}
+method = "{{ output.method|default("POST") }}"
```

Comment:
It's not related to your patch, but I don't think we should have default in the toml templates. We can fix this later if we agree with this principle.